src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -127,7 +127,7 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     }
     const eventReference = {...event};
     if (eventReference.id) {
-      delete eventReference.id;
+      delete (eventReference as any).id;
```

Comment:
What about refactoring the spread to something like 

`{id:_, ...eventReference} = event`

Reply:
eventReference is passed to functions that then expect Event type. Without the id it errors as well. Could change the type of Event but that also feels like a bigger change.